assets/src/apps/authoring/components/EditingCanvas/EditingCanvas.tsx

```diff
@@ -126,6 +132,39 @@ const EditingCanvas: React.FC = () => {
     dispatch(setRightPanelActiveTab({ rightPanelActiveTab: RightPanelTabs.SCREEN }));
   }, [currentActivityId]);
 
+  useKeyDown(
+    () => {
+      if (currentSelectedPartId && !configPartId?.length && _currentPartPropertyFocus) {
+        setNotificationStream({
+          stamp: Date.now(),
+          type: NotificationType.CHECK_SHORTCUT_ACTIONS,
+          payload: { id: currentSelectedPartId, type: 'Delete' },
+        });
+      }
+    },
+    ['Delete', 'Backspace'],
+    {},
+    [currentSelectedPartId, configPartId, _currentPartPropertyFocus],
```

Comment:
Adding the `useKeyDown()` here was only calling the delete part once. To notify the component that has the delete code, I am using the notification pusher.

Note: @bsparks you can recommend the notification type name :)